include/of_controller.h

```diff
@@ -56,8 +63,12 @@ class OFController : public OFServer {
             switch_dpid_map(switch_dpid_map),
             port_id_map(port_id_map),
             OFServer(address, port, nthreads, secure,
-                     OFServerSettings().supported_version(4) // OF version 0x04 is OF 1.3
-                         .echo_interval(30)) { }
+                     OFServerSettings()
+                         .supported_version(4) // OF version 1 is OF 1.3
```

Comment:
"OF version 1 is OF 1.0 and version 4 is 1.3"

Reply:
fixed